### PR TITLE
Feature screens and camera

### PR DESCRIPTION
**Pull request by Tclv**
Refactored ElevatorMusic to now instantiate a screen. This screen is GameScreen and contains most of the temporary code that ElevatorMusic contained first. This screen also implements a viewport and fixes a bug in the old camera behaviour. This bug was that it rendered coordinates not appearing in the screen (for the time being hardcoded to always render >0 coordinates and the camera didnt properly update on player movement.

### CODE REVIEW

File: core/src/com/elevatormusic/gui/GameScreen.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.Vector3;
+import com.badlogic.gdx.utils.viewport.FitViewport;
+import com.elevatormusic.gameobject.GameObject;
+import com.elevatormusic.gameobject.driver.DriverComponent;
+import com.elevatormusic.gameobject.physics.PhysicsComponent;
+import com.elevatormusic.gameobject.renderer.RendererComponent;
+import com.elevatormusic.renderer.Room;
+
+public final class GameScreen extends ScreenAdapter {
```
Comment (jonascarpay) [inline core/src/com/elevatormusic/gui/GameScreen.java]: Javadoc met verantwoordelijkheden?

**Comment by Tclv** [comment]
Done

File: core/src/com/elevatormusic/ElevatorMusic.java [hunk 2]
```diff
@@ -1,74 +1,16 @@
...
-import com.elevatormusic.gameobject.physics.PhysicsComponent;
-import com.elevatormusic.gameobject.renderer.RendererComponent;
-import com.elevatormusic.renderer.Room;
+import com.badlogic.gdx.Game;
+import com.elevatormusic.gui.GameScreen;
 
 
 /**
- * Main class for the core component of the game.
+ * Main class of the game.
```
Comment (jonascarpay) [inline core/src/com/elevatormusic/ElevatorMusic.java]: Verantwoordelijkheden?

File: core/src/com/elevatormusic/gui/GameScreen.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+import com.elevatormusic.gameobject.physics.PhysicsComponent;
+import com.elevatormusic.gameobject.renderer.RendererComponent;
+import com.elevatormusic.renderer.Room;
+
+/**
+ * Screen that concerns itself with the conversion
+ * from screen units to world units. Also distributes
+ * the render calls to the various components. Has some
+ * extra responsibilities currently (world instantiation
+ * amongst others) which should be refactored when more advanced
```
Comment (jonascarpay) [inline core/src/com/elevatormusic/gui/GameScreen.java]: je moeder heeft advanced behaviour

**Pull request merged**